ONSdigital/ras-secure-message
# Fix vulnerable packages

# What and why?

There was a vulnerability in our packages.  This PR bumps everything possible to the latest version to fix it.  One thing that we couldn't bump past a certain version was Flask.  When it's set to =>2.2.x then the behave tests fail.  We can address this in another card specifically.

# How to test?

- Deploy app
- Run acceptance tests
- Use the secure message functionality yourself a bit.  It should all work

# Trello
